src/tracking/envelope.cpp

```diff
@@ -77,7 +78,7 @@ namespace impactx
         amrex::ParmParse const pp_algo("algo");
         bool space_charge = false;
         pp_algo.query("space_charge", space_charge);
-        AMREX_ALWAYS_ASSERT_WITH_MESSAGE(!space_charge, "Space charge not yet implemented for envelope tracking.");
+        //AMREX_ALWAYS_ASSERT_WITH_MESSAGE("2D space charge only is implemented for envelope tracking.");
```

Comment:
Ah, maybe a good time for us to introduce a new parameter for the space charge variants we will implement soon and in the future (2D, 3D, other?).

Something like `algo.space_charge_model = ...` (string?)

Reply:
Agreed.